query/queries_large.ipynb

```diff
@@ -0,0 +1,1489 @@
+{
```

Comment:
1. Can we also add a brief note on how to create the SQL-on-FHIR V2 views using the pipeline. 
2. Question: Should we mention the performance numbers for the SQL-on-FHIR V2 queries on PostgreSQL?

Reply:
Re. 1: The Prerequisites section refers to `queries_and_views` notebook which has some set up notes; those are based on the controller and SQL-on-FHIR-v2 views are not added there yet. I'll update this once the controller is updated.

Re. 2: I added a section at the end.